### PR TITLE
deprecate RangeEnabled, but leave in API

### PR DESCRIPTION
**Pull request by jaffee**
## Overview

the RangeEnabled option now has no effect, but it still exists in the API. A few
tests still use it to ensure this. This would only be considered a breaking
change if someone was relying on Pilosa to enforce the RangeEnabled: false
option to prevent fields being created in certain frames. This seems unlikely.

Fixes part of #1194 - will need to finish deprecation after 0.9 release.

## Pull request checklist

- [ ] I have read the [contributing guide](https://github.com/pilosa/pilosa/blob/master/CONTRIBUTING.md).
- [ ] I have agreed to the [Contributor License Agreement](https://cla-assistant.io/pilosa/pilosa).
- [ ] I have updated the [documentation](https://github.com/pilosa/pilosa/tree/master/docs).
- [ ] I have resolved any merge conflicts.
- [ ] I have included tests that cover my changes.
- [ ] All new and existing tests pass.

## Code review checklist
This is the checklist that the reviewer will follow while reviewing your pull request. You do not need to do anything with this checklist, but be aware of what the reviewer will be looking for.

- [ ] Ensure that any changes to external docs have been included in this pull request.
- [ ] If the changes require that minor/major versions need to be updated, tag the PR appropriately.
- [ ] Ensure the new code is [properly commented](https://github.com/golang/go/wiki/CodeReviewComments#doc-comments) and follows [Idiomatic Go](https://dmitri.shuralyov.com/idiomatic-go).
- [ ] Check that tests have been written and that they cover the new functionality.
- [ ] Run tests and ensure they pass.
- [ ] Build and run the code, performing any applicable integration testing.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - Group the write operations in syncBlock by MaxWritesPerRequest ([#950](https://github.com/pilosa/pilosa/pull/950))
 - Refactored HTTPClient handling ([#991](https://github.com/pilosa/pilosa/pull/991))
 - Remove FrameSchema. Move Fields to the Frame struct ([#907](https://github.com/pilosa/pilosa/pull/907))
+- Deprecated RangeEnabled option. ([#1205](https://github.com/pilosa/pilosa/pull/1205))
```
Comment (codysoyland) [inline CHANGELOG.md]: Remove period for consistency

File: index_test.go [hunk 2]
```diff
@@ -124,14 +124,14 @@ func TestIndex_CreateFrame(t *testing.T) {
 			}
 		})
 
-		t.Run("ErrInverseRangeNotAllowed", func(t *testing.T) {
+		t.Run("ErrInverseRangeAllowed", func(t *testing.T) {
```
Comment (codysoyland) [inline index_test.go]: How does a frame with inverse and range enabled work? Trying to understand why it wasn't allowed before.

Comment (jaffee) [reply index_test.go]: I've been looking into this - the fields, and the inverse and standard views are handled separately, so the inverse setting does not affect fields. I'll add a basic test, but I'm not going to spend much time on it until we decide what we're doing with #1195

Comment (jaffee) [reply index_test.go]: OK, we decided to deprecate rangeEnabled - is this cool now?

**Pull request merged**